jbmc/src/java_bytecode/java_bytecode_parse_tree.h

```diff
@@ -164,10 +164,11 @@ struct java_bytecode_parse_treet
 
     void output(std::ostream &out) const;
 
-    methodt():
-      is_native(false),
-      is_abstract(false),
-      is_synchronized(false)
+    methodt()
+      : is_native(false),
+        is_abstract(false),
+        is_synchronized(false),
+        is_bridge(false)
```

Comment:
This is something I'll have to remind Thomas about, but setting class fields like that is prone to errors. It's much better idea to define their default values in the class definition, like so:


```c++
private:
  bool is_native = false;
  bool is_abstract = false;
  bool is_synchronized = false;
  bool is_bridge = false;
```

That way, when you decide to define extra constructors, there's no risk of ever forgetting to initialize values.